assets/_sass/components/_social-share-icons.scss

```diff
@@ -0,0 +1,69 @@
+/*==================================
+=       Social Share Icons         =
+==================================*/
+
+.share {
+    $s: &;
+    $i: icon-facebook, icon-twitter, icon-email, icon-print;
```

Comment:
This is clever, but there's no need for this. You can select the `i` directly like you did on line 32.